FreeRTOS/FreeRTOS
# Smp test cases

Push SMP Unit Test cases

Description
-----------
Push SMP Unit Test cases after separating them from the framework

Test Steps
-----------
Overall coverage rate:
  lines......: 97.1% (2873 of 2958 lines)
  functions..: 100.0% (216 of 216 functions)
  branches...: 92.0% (1699 of 1847 branches)

Checklist:
----------


- [ x] I have tested my changes. No regression in existing tests.
- [x ] I have modified and/or added unit-tests to cover the code changes in this Pull Request.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
